### ISSUE TITLE
Título do PDF

### ISSUE THREAD

**Issue by dubbows** [nfephp-org/sped-da#602]
Para incluir títulos no PDF é necessário essa implementação:

vendor/nfephp-org/sped-da/src/NFe/Danfe.php

class Danfe extends DaCommon
{
...
   protected $title = '';

   public function setTitle($title)
   {
      $this->title = $title;
   }

   protected function monta()
   {
     ...
      $this->pdf->setTitle($this->title);
     ...
    }
...
}

Se puder incluir a implementação seria de grande valia.

**Comment by gustavokennedy** [comment]
Olá @dubbows! 

Abri um [PR com a solicitação.](https://github.com/nfephp-org/sped-da/pull/625)

**Comment by f3lpz** [comment]
Boa tarde @gustavokennedy , esse pull request quebrou a impressão, pois na linha 500 esta atribuindo o valor da variavel pdf uma string vazia, isso da o erro ` Call to a member function setTitle() on string`

**Comment by gustavokennedy** [comment]
Verdade, vou incluir o `setTitle` na linha 513, abaixo da instância `$this->pdf = new Pdf($this->orientacao, 'mm', $this->papel);`.

**Comment by gustavokennedy** [comment]
Corrigido! @f3lpz @dubbows

**Comment by f3lpz** [comment]
Valeu @gustavokennedy !

**Comment by robmachado** [comment]
@gustavokennedy sempre faça um teste local antes de subir um PR, como erá uma coisa muito simples nem me atentei ao local onde estava a implementação e autorizei (falha minha por não conferir)

**Comment by gustavokennedy** [comment]
@robmachado Pode deixar! Foi tão simples que deixei passar também! Nas próximas farei, com certeza. 👏👏

**Comment by gustavokennedy** [comment]
@robmachado Lembrei agora também! Não teriamos que alterar nada nas Documentações?

**Comment by robmachado** [comment]
Poes é a documentação está "paupérrima" falta tudo kkkk vide https://github.com/nfephp-org/sped-da/blob/master/docs/Danfe.md

**Comment by robmachado** [comment]
Eu estou fazendo uma refatoração da Danfe, por hora estou chamando de Danfe2, estou movendo as (ou a maioria das)  estruturas para traits assim fica mais facil manter o codigo em blocos menores
